Services/LoggerService.php

```diff
@@ -13,21 +13,62 @@
 
 class LoggerService
 {
-    const LOG_LEVEL = 'warning';
+    const SHOPWARE_LOG_LEVEL = 'warning';
+
+    const NLX_LOG_LEVEL_BASIC = 'BASIC';
+    const NLX_LOG_LEVEL_INFO = 'INFO';
+    const NLX_LOG_LEVEL_DEBUG = 'DEBUG';
+
+    const NLX_LOG_LEVELS = [
```

Comment:
I still would use the standard PSR log levels here
https://www.php-fig.org/psr/psr-3/